src/main/proto/Report.proto

```diff
@@ -60,4 +65,24 @@ message Schedule {
 message ReportLog {
     string task_status = 1;
     string task_executed = 2;
+    string threshold_met = 3;
+    string notification_sent = 4;
+    string channel = 5;
+}
+
+message ChannelConnection {
+    repeated Channelparameters ChannelProperties = 1;
+}
+
+message Channelparameters {
```

Comment:
should be camelCase for consistency `ChannelParameters`

Reply:
okay